device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceClient.java

```diff
@@ -173,7 +173,7 @@ public DeviceClient(String connString, TransportClient transportClient) throws U
     public DeviceClient(String connString, IotHubClientProtocol protocol) throws URISyntaxException, IllegalArgumentException
     {
```

Comment:
Just make this constructor call your new constructor but with a null client options. No need to call commonConstructorVerifications/setup here and in your new constructor

Reply:
Agreed